### DISCUSSION TITLE
Crazyflie Bolt 1.1 Stuck on Solid Blue M2 LED

### DISCUSSION THREAD

**Discussion by bhabas** [bitcraze/discussions#1199] [Q&A]
I've recently been using a Crazyflie Bolt 1.1 on a custom 4S (15.2V-16.8V) quadcopter without any issues. However, after an initial boot-up test on our new 6S system with an ESC that powers the Bolt through a 10V/2A BEC, it doesn't want to function properly anymore.

Here's what I've noticed and tried:
- In firmware mode, only the blue M2 LED comes on and none of the remaining LEDs flash in the normal startup pattern
- When I try to connect via cfclient, scanning and connecting doesn't work but I can upload new firmware via the bootloader and then still have the same issue.  
- When powered by a 4S battery none of the chips are getting hot so I can't tell if something was shorted or fried
- Connecting via micro-usb will power the board but cfclient can't identify it (this does work with another working Bolt we have)
- I am able to detect the malfunctioning Bolt via USB via it's [DFU mode](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/dfu/#bootloader-recovery)
- I also double checked the voltage that initially powered the Bolt and it was the expected 10V

I'm kind of at a loss for what to try, and was curious if you have any ideas on how to recover from this or what might have happened?

**Comment by tobbeanton** [comment]
My initial guess would be that he STM32 got fried somehow but since you say it works in DFU mode it gets more complicated and perhaps it is partially damaged. Could also be that something is preventing the firmware from starting. Do you have access to a debugger. That would probably be the fastest way of figuring out what is wrong, assuming it is possible to connect to the STM32.

**Comment by matejkarasek** [comment]
Randomly stumbled on this. We have seen this behaviour lately on a couple of Bolts. The solution was to flash it with a much older firmware via cfclient (so both the STM and NRF firmware get updated), which brought them alive. Updating to the newest firmware later worked fine. 

Could it be that some there exists some "bad" combination STM and NRF firmware versions causing this? We often keep updating the STM firmware but NRF may stay at an older version...

**Reply by ataffanel** [reply]
This is also the solution we observed worked for us. It seems that new client have problem detecting what softdevice is flashed and so do not update it according to what the firmware expects. We will have a look at it, I created a ticket in the client repos: https://github.com/bitcraze/crazyflie-clients-python/issues/811
